src/theory/theory_inference_manager.h

```diff
@@ -132,6 +132,10 @@ class TheoryInferenceManager
    * only be called if there is not yet proof support in the given theory.
    */
   void conflict(TNode conf);
+  /** The number of conflicts we have added since the last call to reset */
```

Comment:
I don't think I like these methods: TheoryState currently says whether we are in conflict, this makes some confusion about what is the right place to ask.

Reply:
Hm, I agree. I've removed them and check whether this has any consequences...

Reply:
Still works.